CognitiveConcierge-iOS/CognitiveConcierge/CognitiveConcierge.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+        <key>ConversationIamApiKey</key>
+        <string>your_key_here</string>
 	<key>ConversationPassword</key>
 	<string>Vzah3JxwdpuI</string>
```

Comment:
blerg. i didn't realize someone checked in values here... we should remove them all.

Reply:
Alright, I'll do that.